stories/Table.story.jsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import styled from "styled-components";
+import { storiesOf } from "@storybook/react";
+import { Table, Col, Row, Icon, AnchorButton } from "../src/index";
+
+const LayoutStyles = styled.div`
+  width: 80%;
+  margin: 0 auto;
+  padding: 100px;
+`;
+
+const medicalPersonsData = [
+  {
+    name: "Alexander Fleming",
+    birthday: "6 August 1881",
+    knownFor: "Discovery of penicillin",
+    wikiLink: "https://en.wikipedia.org/wiki/Alexander_Fleming"
+  },
+  {
+    name: "Edward Jenner",
+    birthday: "17 May 1749",
+    knownFor: "Pioneer of the smallpox vaccine",
+    wikiLink: "https://en.wikipedia.org/wiki/Edward_Jenner"
+  },
+  {
+    name: "William Osler",
+    birthday: "12 July, 1849",
+    knownFor: "Created the first specialty training for physicians",
+    wikiLink: "https://en.wikipedia.org/wiki/William_Osler"
+  }
+];
+
+storiesOf("Table", module)
+  .add("basic", () => {
+    const columns = [
+      {
+        Header: "Name",
+        accessor: "name"
+      },
+      {
+        Header: "Birthday",
+        accessor: "birthday"
+      },
+      {
+        Header: "Known for",
+        accessor: "knownFor"
+      }
+    ];
+
+    return (
+      <LayoutStyles>
+        <Table data={medicalPersonsData} columns={columns} />
+      </LayoutStyles>
+    );
+  })
+  .add("auto sizing column widths", () => {
+    const columns = [
+      {
+        Header: "Name",
+        accessor: "name",
+        maxWidth: 150
+      },
+      {
+        Header: "Birthday",
+        accessor: "birthday",
+        maxWidth: 150
+      },
+      {
+        Header: "Known for",
+        accessor: "knownFor"
+      }
+    ];
+
+    return (
+      <LayoutStyles>
+        <Table data={medicalPersonsData} columns={columns} />
+      </LayoutStyles>
+    );
+  })
+  .add("custom cell contents", () => {
+    // eslint-disable-next-line react/prop-types
```

Comment:
i think i might just define those prop-types. dont have a strong opinion tho.

Reply:
I opted not to because it's just an anonymous component I'm using immediately, proptypes seems unnecessary

Reply:
yeah okay.